app/views/subjects/_subject_with_tailwind.html.erb

```diff
@@ -0,0 +1,12 @@
+<li class="flex gap-x-4 px-3 py-5 justify-between items-center">
```

Comment:
Perhaps we could use a little bit less of padding here? Seems that there's a lot of space between the items. I think I like it tho, at least more than the current cluttered version that we have 🤔